app/web_conn.go

```diff
@@ -73,6 +78,7 @@ func (a *App) NewWebConn(ws *websocket.Conn, session model.Session, t goi18n.Tra
 	return wc
 }
 
+// Close closes the WebConn.
 func (wc *WebConn) Close() {
 	wc.WebSocket.Close()
 	wc.closeOnce.Do(func() {
```

Comment:
Do we really need this `closeOnce`? My understanding is that when we close the ws connection, `readPump()` is going to return and we'll close `endWritePump` in `Pump()`.